src/riak_core_console.erl

```diff
@@ -752,6 +753,27 @@ commit_staged([]) ->
                       "plan and try to commit again~n")
     end.
 
+timezone([]) ->
+    TZ = riak_core_metadata:get({riak_core, timezone}, string,
+                                [{default, "not configured"}]),
+    io:format("~s~n", [TZ]);
+timezone([TZString]) ->
+    TZ = jam:process(jam_iso8601:parse_tz(TZString)),
+    attempt_set_timezone(jam:is_valid(TZ), TZ, TZString).
```

Comment:
That would crash the console, right? That might not be the behavior we want.  It would be nice from a function-perspective, however.